### ISSUE TITLE
Accessibility review and fixes for uids_base superfish

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#1120]
As @inspector508 noted in https://github.com/uiowa/uids/issues/55, there are a number of fixes that need to be made to the toggle-menu option of the superfish menu. 

### Todd's notes:

a.sf-depth-1 menuparent ()

1. Can these use role=”button”?
* aria-haspopup=”true” 
* aria-expanded (toggle) 

### Additional Notes

1. <nolink> creates duplicate empty links in the sf-accordion

<img width="904" alt="Screen Shot 2020-02-27 at 4 24 35 PM" src="https://user-images.githubusercontent.com/1036433/75492750-056cf700-597e-11ea-8640-499483834d03.png">

**Comment by joewhitsitt** [comment]
Semi-related https://github.com/uiowa/uiowa/issues/3406

**Comment by bspeare** [comment]
Related Drupal.org issue: https://www.drupal.org/project/superfish/issues/3107671.

**Comment by joewhitsitt** [comment]
Keyboard navigation items have been split off here: https://github.com/uiowa/uiowa/issues/3517

**Comment by richardbporter** [comment]
We noticed that SF does not clone the menuparent on the tab event which means keyboard navigation users cannot get to the actual link. Trying to fix in #3517.
